### DISCUSSION TITLE
Filter Dropdown grid style

### DISCUSSION THREAD

**Discussion by wit3** [rappasoft/laravel-livewire-tables#520] [Feature Requests]
Hi, in a future release can you display the filter popup dropdown with grid stile?
Because if you have many filters the popup it's to long, i think a grid layout solve the problem.

Thanks

**Comment by sawirricardo** [comment]
This. I have like > 100 options in multiselect. cc @rappasoft

**Reply by nafil161** [reply]
This is released in v2.7.0
